change/@fluentui-react-provider-c610d38f-97b5-4686-ab4d-ec59b20d764a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fix(react-provider): fix lint issues",
+  "packageName": "@fluentui/react-provider",
+  "email": "martinhochel@microsoft.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
I don't see any changes in `react-provider`, is this change file relevant? Or can we just delete it?

Reply:
lemme double check, i might missed something as I was tweaking the changes of both plugin setup and actual changes

Reply:
removed, ty!